setup.py

```diff
@@ -0,0 +1,28 @@
+from distutils.core import setup
+setup(
+  name = 'swmr_tools',         # How you named your package folder (MyLib)
+  packages = ['swmr_tools'],   # Chose the same as "name"
+  version = '0.3',      # Start with a small number and increase it with every change you make
+  license='MIT',        # Chose a license from here: https://help.github.com/articles/licensing-a-repository
+  description = 'Python iterator for safely monitoring nexus files',   # Give a short description about your library
+  author = 'Diamond Light Source Ltd',                   # Type in your name
+  author_email = 'scientificsoftware@diamond.ac.uk',      # Type in your E-Mail
+  url = 'https://github.com/DiamondLightSource/python-swmrtools',   # Provide either the link to your github or to your website
+  download_url = 'https://github.com/rparke/Iterator/archive/v0.1.tar.gz',    # I explain this later on
```

Comment:
Out of date URL

Reply:
Changed to what I think the url will be once we've created a tag. Just need to double check once that's been created.